src/asyncChatGPT/asyncChatGPT.py

```diff
@@ -242,22 +241,10 @@ def refresh_session(self) -> Exception:
 
         :return: None or Exception
         """
-        if (
-            "session_token" not in self.config
-            and ("email" not in self.config or "password" not in self.config)
-            and "Authorization" not in self.config
-        ):
-            error = ValueError("No tokens provided")
-            self.debugger.log(error)
-            raise error
-        elif "session_token" in self.config:
-            if (
-                self.config["session_token"] is None
-                or self.config["session_token"] == ""
-            ):
-                raise ValueError("No tokens provided")
+        # Either session_token, email and password or Authroization is required
+        if self.config.get("session_token"):
```

Comment:
Missing check for email/password or authorization? At least one of the methods is needed

Reply:
Ah I see. It goes to else automatically